### DISCUSSION TITLE
built in wifi blocker

### DISCUSSION THREAD

**Discussion by siouxxsies** [3kh0/ext-remover#581] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
apparently my school's wifi has a restriction and blocks website still even when i disabled extensions.
can anyone help me out here? crying rn

**Comment by qDread-dev** [comment, marked as answer]
use a vpn

**Reply by siouxxsies** [reply]
how exactly would i download one

**Reply by qDread-dev** [reply]
You could follow this method (from in TN discord):
Download an onc file listed below, head over to chrome://network and scroll all the way down until you see something like this:
https://media.discordapp.net/attachments/1059525655622070322/1059526197891059712/image.png
Upload your downloaded onc file, and then click alt shift s. On the second page of it, you will find VPN there which you can then click and connect to ProtoVPN. If you somehow need an alternative you can go to chrome://network#/select, and find the vpn there. This can also help if chrome://os-settings is blocked for you.
If chrome://network, you can also use chrome://os-settings, and manually click add built in vpn, then make a vpn with these settings: 
https://media.discordapp.net/attachments/1059525655622070322/1059526294582333532/image.png

CREDIT:
Thanks for @The Greatest Giant for setting up the VPS, (Virtual Private Server) and thanks for the oracle account created by @Mr_voidMC!
They both should unblock websites just fine!

https://cdn.discordapp.com/attachments/1059377423734538280/1059525175898550373/ProtoVPN.onc
https://cdn.discordapp.com/attachments/1059377423734538280/1059526664163446804/GiantVPN.onc

**Reply by siouxxsies** [reply]
> chrome://network#/select

it just says connect failed, that's all,  it also pops up the configuration window for the vpn too

**Reply by qDread-dev** [reply]
> > chrome://network#/select
> 
> it just says connect failed, that's all, it also pops up the configuration window for the vpn too

🤷

**Reply by siouxxsies** [reply]
> > > chrome://network#/select
> > 
> > 
> > it just says connect failed, that's all, it also pops up the configuration window for the vpn too
> 
> 🤷

nevermind !! I just gotta use my hotspot thats all, thanks man

**Reply by siouxxsies** [reply]
> > > chrome://network#/select
> > 
> > 
> > it just says connect failed, that's all, it also pops up the configuration window for the vpn too
> 
> 🤷

also do you have a TN discord server invite maybe? thanks

**Reply by qDread-dev** [reply]
> > > > chrome://network#/select
> > > 
> > > 
> > > it just says connect failed, that's all, it also pops up the configuration window for the vpn too
> > 
> > 
> > 🤷
> 
> also do you have a TN discord server invite maybe? thanks
https://discord.gg/unblock

**Reply by JmarbinGG** [reply]
> You could follow this method (from in TN discord):
> 
> Download an onc file listed below, head over to chrome://network and scroll all the way down until you see something like this:
> 
> https://media.discordapp.net/attachments/1059525655622070322/1059526197891059712/image.png
> 
> Upload your downloaded onc file, and then click alt shift s. On the second page of it, you will find VPN there which you can then click and connect to ProtoVPN. If you somehow need an alternative you can go to chrome://network#/select, and find the vpn there. This can also help if chrome://os-settings is blocked for you.
> 
> If chrome://network, you can also use chrome://os-settings, and manually click add built in vpn, then make a vpn with these settings: 
> 
> https://media.discordapp.net/attachments/1059525655622070322/1059526294582333532/image.png
> 
> 
> 
> CREDIT:
> 
> Thanks for @The Greatest Giant for setting up the VPS, (Virtual Private Server) and thanks for the oracle account created by @Mr_voidMC!
> 
> They both should unblock websites just fine!
> 
> 
> 
> https://cdn.discordapp.com/attachments/1059377423734538280/1059525175898550373/ProtoVPN.onc
> 
> https://cdn.discordapp.com/attachments/1059377423734538280/1059526664163446804/GiantVPN.onc 
> 
> 



could u pls update the image links for how to config the vpn? when i try to view the images, it makes me download them and they are empty, it just says "0 bytes"

**Comment by abditube1345** [comment]
no it no working because its no way to bypass this use skiovox
